.github/workflows/test.yml

```diff
@@ -0,0 +1,65 @@
+name: Test
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ main, develop ]
+  pull_request:
+    branches: [ main, develop ]
+
+jobs:
+  test-ubuntu-20:
+    name: Run tests on Ubuntu 20.04
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.8", "3.9", "3.10"]
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
```

Comment:
`actions/checkout@v3` for new CI should be a bit more future-proof as Github Actions moves everything to Node 16